src/components/NavBar.jsx

```diff
@@ -1,15 +1,27 @@
+
 import React from 'react';
-import { AppBar, Toolbar, Typography } from '@material-ui/core';
+import { AppBar, Button, Grid, Typography } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+
 
 const NavBar = () => {
-  return (
+  return(
     <div>
       <AppBar position="static">
-        <Toolbar>
-          <Typography variant="title" color="inherit">
-            YouTube Clone Logo
-          </Typography>
-        </Toolbar>
+        <Typography variant="display1" color="inherit">
+          <Grid container sm={12} justify="center" alignItems="center">
```

Comment:
this line is returning error as below.
```
Warning: Failed prop type: The property `sm` of `Grid` must be used on `item`.
```
it hasn't shown after I have tried deleting `sm={12}`.